medusa/clients/torrent/utorrent_client.py

```diff
@@ -75,17 +123,33 @@ def _add_torrent_file(self, result):
         )
 
     def _set_torrent_label(self, result):
+        torrent_new_label = result.series.name
+
         if result.series.is_anime and app.TORRENT_LABEL_ANIME:
             label = app.TORRENT_LABEL_ANIME
         else:
             label = app.TORRENT_LABEL
 
-        return self._request(params={
-            'action': 'setprops',
-            'hash': result.hash,
-            's': 'label',
-            'v': label,
-        })
+        log.info('torrent label was {path}',
+                    {'path': label})
+
+        label = label.replace("%N",torrent_new_label)
```

Comment:
Single quotes, spaces, etc. This won't pass the tests otherwise.

Reply:
Done. I hope I covered all of those.